docs/static/plugin-manager.asciidoc

```diff
@@ -112,6 +112,21 @@ bin/logstash-plugin update logstash-input-github <2>
 <1> updates all installed plugins
 <2> updates only the plugin you specify
 
+[NOTE]
```

Comment:
Preview: https://logstash_bk_16899.docs-preview.app.elstc.co/guide/en/logstash/master/working-with-plugins.html#updating-plugins

I added this as an admonition because in the normal case users shouldn't need to concern themselves with this and will be happy to get the latest _fully-compatible_ version available. It is rare that a user would _want_ to jump across major versions of a plugin because we are taking care to backport non-breaking features and fixes to the previous minor.